aea/skills/base.py

```diff
@@ -510,6 +510,17 @@ def parse_module(  # pylint: disable=arguments-differ
         """
         return _parse_module(path, handler_configs, skill_context, Handler)
 
+    def protocol_dialogues(self, attribute: Optional[str] = None):  # type: ignore
```

Comment:
> type: ignore

Because of the circular import caused by `Dialogue` class cannot assign a return type to this